### DISCUSSION TITLE
Tensorflow requirements

### DISCUSSION THREAD

**Discussion by kpasad** [NVlabs/sionna#72] [Q&A]
For sionna 0.11.0, Using tensorflow 2.10.0 shows the error:
"ModuleNotFoundError: No module named 'tensorflow.python.trackable'"

Per the documentation Tensorflow version between 2.6 and 2.11 are supported.
But the requirements.txt says "tensorflow!=2.7.0, !=2.7.1, !=2.8.0, <2.11 and >=2.6.4;"

what is the recommended version?

**Comment by gmarcusm** [comment]
Can you please provide the version of keras installed and the OS version you use?
